spec/controllers/admin/events_controller_spec.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+require './spec/rails_helper'
+
+RSpec.describe Admin::EventsController, type: :controller do
```

Comment:
This test verifies the Pundit behavior - if the user is an admin, the endpoint should return 200. Otherwise, it returns 401.